components/kubeflow-pipeline/pipeline_components/analytics-client/Makefile

```
@@ -5,18 +5,23 @@ MAKEVAR_REGISTRY?=ghcr.io/katulu-io/fl-suite
 CONTAINER_TAG = $(shell echo "${MAKEVAR_VERSION}" | sed "s/\+[0-9a-zA-Z-]*//")
 
 lint:
-	@echo "lint not implemented"
+	poetry install --no-root
+	poetry run python -m isort --check-only flwr_analytics_client/
```

Unless there is a specific reason, I would encourage to use the short version `poetry run isort ...`. This applies to all the other instances where `poetry run` is used:

```suggestion
	poetry run isort --check-only flwr_analytics_client/
```

---

By starting these tools from `python`, the current directory gets added to `PYTHONPATH`. This is helpful for some tools (e.g. `pytest`) to help figure out the right import paths. While tools like `isort` and `black` shouldn't be affected, let's just use this pattern all the time.